registry/store/external_registry.go

```diff
@@ -24,18 +24,18 @@ import (
 type ExternalRegistry interface {
 	ReadKeys(namespace auth.Namespace) ([]string, error)
 	ReadServiceInstanceByInstID(namespace auth.Namespace, instanceID string) (*ServiceInstance, error)
-	ListServiceInstancesByKey(namespace auth.Namespace, key string) (map[string]*ServiceInstance, error)
+	ListServiceInstancesByName(namespace auth.Namespace, name string) (map[string]*ServiceInstance, error)
 	ListAllServiceInstances(namespace auth.Namespace) (map[string]ServiceInstanceMap, error)
 	InsertServiceInstance(namespace auth.Namespace, instance *ServiceInstance) error
-	DeleteServiceInstance(namespace auth.Namespace, key string) (int, error)
+	DeleteServiceInstance(key string) (int, error)
```

Comment:
Sorry, forgot to review the 2nd commit, which already corrected the namespace parameter.